app/views/spina/conferences/primer_theme/conferences/show.html.haml

```diff
@@ -5,11 +5,11 @@
         - cache partable_for(:submission_url, parent: @conference) do
           - if @conference.content(:submission_url).present?
             = link_to t(:'.submit_abstract'), @conference.content(:submission_url),
-                      class: %i[btn btn-sm flash-action d-none d-sm-inline-block]
+                      class: %i[btn btn-sm flash-action p-3 p-sm-1]
         - cache partable_for(:submission_date, parent: @conference) do
           %p
```

Comment:
I'm guessing this is to get around the wrapping problem. If the text is `display: inline-block`, then the `flash-action` should float above it more readily, and the button's padding doesn't have to be reduced.
```suggestion
                      class: %i[btn btn-sm flash-action mb-2]
        - cache partable_for(:submission_date, parent: @conference) do
          %p.d-inline-block
```

Reply:
Wrapping isn't really a problem though even with small screens, I just wanted to make the button a bit more obvious - I'll remove the padding and switch to `btn-primary` which I think will be fine.